accounts/abi/bind/base.go

```diff
@@ -231,13 +235,33 @@ func (c *BoundContract) transact(opts *TransactOpts, contract *common.Address, i
 	} else {
 		rawTx = types.NewTransaction(nonce, c.address, value, gasLimit, gasPrice, input)
 	}
+
+	return rawTx, nil
```

Comment:
I can't comment on line 231. It said "// Create the transaction, sign it and schedule it for execution".

Here since this method only creates the rawTx, suggest to modify it to "Create the raw transaction"